pkg/abi/linux/fuse.go

```diff
@@ -247,6 +263,63 @@ type FUSEInitOut struct {
 	_ [8]uint32
 }
 
+// FUSEInitRes is a variable-length wrapper of FUSEInitOut. The FUSE server may
+// implement older version of FUSE protocol, which contains a FUSEInitOut with
+// less attributes.
+//
+// Dynamically-sized objects cannot be marshalled.
+type FUSEInitRes struct {
+	marshal.StubMarshallable
+
+	// InitOut contains the response from the FUSE server.
+	InitOut FUSEInitOut
+
+	// Len is the total length of bytes of the response.
+	Len uint32
+}
+
+// UnMarshalBytes deserializes src to the InitOut attribute in a FUSEInitRes.
+func (r *FUSEInitRes) UnmarshalBytes(src []byte) {
```

Comment:
No. Because here we're dealing with a variable length of FUSEInitOut. Any fixed-size unmarshalling method fails unless we declare a subset of FUSEInitOut that only contains the attributes known by the lowest supported FUSE version.